src/hooks.server.ts

```diff
@@ -8,9 +8,14 @@ import { ADMIN_ROLE } from './utilities/permissions';
 
 export const handle: Handle = async ({ event, resolve }) => {
   if (env.PUBLIC_LOGIN_PAGE === 'disabled') {
-    const permissibleQueries = await effects.getUserQueries('');
-    event.locals.user = { allowedRoles: [ADMIN_ROLE], defaultRole: ADMIN_ROLE, id: 'unknown', token: '' };
-    event.locals.permissibleQueries = permissibleQueries ?? {};
+    const permissibleQueries = await effects.getUserQueries(null);
+    event.locals.user = {
+      allowedRoles: [ADMIN_ROLE],
+      defaultRole: ADMIN_ROLE,
+      id: 'unknown',
+      permissibleQueries,
```

Comment:
Do you mind keeping the `permissibleQueries ?? {}` for this? It's possible that the query returns `null`.

Reply:
Instead can we make `getUserQueries` just return `{}` instead of `null`? Essentially remove the `null` from the equation?

Reply:
Works for me!